hack/kind-up.sh

```diff
@@ -167,6 +172,88 @@ check_registry_cache_availability() {
   done
 }
 
+# The default StorageClass which comes with `kind' is configured to use
+# rancher.io/local-path (see [1]) provisioner, which defaults to `hostPath'
+# volume (see [2]).  However, `hostPath' does not expose any metrics via
+# kubelet, while `local' (see [3]) does. On the other hand `kind' does not
+# expose any mechanism for configuring the StorageClass it comes with (see [4]).
+#
+# This function annotates the default StorageClass with `defaultVolumeType: local',
+# so that we can later scrape the various `kubelet_volume_stats_*' metrics
+# exposed by kubelet (see [5]).
+#
+# References:
+#
+# [1]: https://github.com/rancher/local-path-provisioner
+# [2]: https://kubernetes.io/docs/concepts/storage/volumes/#hostpath
+# [3]: https://kubernetes.io/docs/concepts/storage/volumes/#local
+# [4]: https://github.com/kubernetes-sigs/kind/blob/main/pkg/cluster/internal/create/actions/installstorage/storage.go
+# [5]: https://kubernetes.io/docs/reference/instrumentation/metrics/
+_setup_kind_sc_default_volume_type() {
```

Comment:
Nit: You could fix the indentation, all of the other funcs use 2 spaces for indentation.